sql/src/main/java/io/druid/sql/calcite/rule/DruidRules.java

```diff
@@ -227,6 +233,42 @@ public void onMatch(final RelOptRuleCall call)
       }
     };
 
+    public static RelOptRule AGGREGATE_SORT_PROJECT = new DruidOuterQueryRule(
```

Comment:
Can you create a test that hits this rule? It should be a nested groupby where the outer query has the sort + project combo.

Reply:
Added a test.